### PR TITLE
Add uninstall instructions #266

### PR DESCRIPTION
**Pull request by jlprat**
Refs: #266
Provides instructions for both Docker and Python installation

# About this change: What it does, why it matters
It matters because a good README should provide means to install and uninstall the product / library.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
...
+  docker stop <CONTAINER_ID>
+
+If you don't need or want to have the Karapace images around you can now proceed to delete them using:
+
+  docker rm <CONTAINER_ID>
+
+Installed from Sources
+----------------------
+
+If you installed Karapace from the sources via ``pithon setup.py install``, it can be uninstalled with the following ``pip`` command:
```
Comment (encima) [inline README.rst:426]: pithon -> python

File: README.rst [hunk 2]
```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
...
+
+  pip uninstall karapace
```
Comment (encima) [inline README.rst:428]: ``pip uninstall karapace``

File: README.rst [hunk 3]
```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
...
+To unistall Karapace from the system you can follow the instructions described below. We would love to hear your reasons for uninstalling though. Feel free to drop us a line letting us know what we can improve!
+
+Installed via Docker
+--------------------
+
+If you installed Karapace via Docker, you would need to first stop and remove the images like described:
+
+First obtain the container IDs related to Karapace, you should have one for the registry itself and another one for the rest interface.
+
+  docker ps | grep karapace
```
Comment (encima) [inline README.rst:413]: also code blocks for these? not only for looks but also for easier copying

File: README.rst [hunk 4]
```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Uninstall
+=========
+
+To unistall Karapace from the system you can follow the instructions described below. We would love to hear your reasons for uninstalling though. Feel free to drop us a line letting us know what we can improve!
```
Comment (encima) [inline README.rst:404]: Maybe something like: 

> Please file an issue if you experience any problems or email [us](support@aiven.io) with feedback

File: README.rst (hunk 2 above)
Comment (jlprat) [reply README.rst:428]: is this the same? or do you mean as a code block?

File: README.rst (hunk 3 above)
Comment (jlprat) [reply README.rst]: Wasn't code snippets 2 tabs?

Comment (jlprat) [reply README.rst]: Got it now

File: README.rst (hunk 2 above)
Comment (encima) [reply README.rst]: my bad, my window was too small to see the indent :facepalm:

**Review by encima — DISMISSED** [review]

**Comment by jlprat** [comment]
Squashed the commits to have a cleaner history

**Review by encima — APPROVED** [review]

**Pull request merged**